static/index.html

```diff
@@ -63,26 +99,78 @@
             window.location.replace(data['url']);
           }
         }).fail(function() {
-          $('#error-unexpected').show();
+          showError('#error-unexpected');
         });
       }
-    </script>
-    <style>
-      .hidden { display: none; }
-      div {
-        position: absolute;
-        left: 50%;
-        top: 50%;
-        transform: translate(-50%, -50%);
+
+      function showError(errorSelector) {
+        showWaitingRoom();
+        $(errorSelector).show();
+        $('#message-please-wait').hide();
+        $('#icon-please-wait').hide();
+        $('#icon-exclamation').show();
       }
-    </style>
+
+      function showWaitingRoom() {
+        $('#consent-ui').hide();
+        $('#waiting-room-ui').show();
+      }
+
+      function setLanguage(languageId) {
+        var languageAssets = getAssetsForLanguage(languageId);
+
+        $('#consent-message').html(languageAssets.consentMessage);
+        $('#consent-ack').text(languageAssets.continueButton);
+        $('#message-please-wait').html(languageAssets.waitingRoomMessage);
+        $('#language-label').html(languageAssets.languageSelect);
+        $('#welcome-message').html(languageAssets.welcomeMessage);
+      }
+    </script>
   </head>
   <body>
-    <div>
-      <p class="hidden" id="error-no-encounter">No encounter was selected</p>
-      <p class="hidden" id="error-unexpected">An unexpected error occurred in the application</p>
-      <p class="hidden" id="error-smart-failed">An error occurred while communicating with the EHR system</p>
-      <p class="hidden" id="message-please-wait">Please wait for the meeting to start...</p>
-    </div>
+    <div class="vertical-center">
+      <div class="middle">
+        <img src="assets/logo.png"
+          class="logo">
+      </div>
+      <div class="middle">
+        <div id="consent-ui" class="top-down consent">
+          <div class="middle">
+          <div id="language-label" class="language-label">Select your language:</div>
+          <select id="language-selector" class="language-selector">
+            <option value="en">English</option>
+            <option value="es">Español</option>
+          </select>
+        </div>
+          <div id="consent-message" class="consent">
+            <h1>Before you enter the waiting room, please review the following information about televisits:</h1>        
+            <ol>
+              <li>Please take a moment to see that you are in a private location where you cannot be unintentionally overheard. If you cannot be in a private location, you can decide at any time whether to continue this visit or to end it.</li>
```

Comment:
Why is the english content duplicated here?

Reply:
I was thinking of perhaps replacing that with a document.ready call at page initialization so we don't have the duplication. I will attempt that and update the PR.